tools/samtools.wdl

```diff
@@ -1327,7 +1327,90 @@ task faidx {
         cpu: 1
         memory: "4 GB"
         disks: "~{disk_size_gb} GB"
-        container: "quay.io/biocontainers/samtools:1.17--h00cdaf9_0"
+        container: "quay.io/biocontainers/samtools:1.19.2--h50ea8bc_0"
```

Comment:
Does this need to be updated for the rest of the file? I thought you added a lint or a CI or something to catch these cases...

Reply:
I think this was just one stray task that didn't get updated. Nothing else uses `1.17--h00cdaf9_0` at the moment.